src/scss/layout/_layout.scss

```diff
@@ -57,17 +43,59 @@ article {
 			display: table;
 		}
 	}
-}
 
-/*
- * Default WP Alignment Classes
+	/*
+ * Default WP image Alignment Classes
  *****************************************************************************/
-figcaption {
-	display: block;
-	font-size: rem(14);
-	font-style: italic;
-	padding: 1rem 0;
-	text-align: center;
+	figure {
+		display: table;
+		margin-left: auto;
+		margin-right: auto;
+	}
+
+	.wp-block-image {
+		img {
+			height: 100%;
+			max-width: 100%;
+		}
+
+		figcaption {
+			caption-side: bottom;
+			display: table-caption;
+			font-size: rem(14);
+			font-style: italic;
+			padding: 1rem 0;
+			text-align: center;
+		}
+
+		.alignleft {
+			float: left;
+			margin-right: rem(25);
+			margin-top: 0;
+		}
+
+		.alignright {
+			float: right;
+			margin-left: rem(25);
+			margin-top: 0;
+		}
+	}
+
+	.wp-block-image figure .alignfull {
+		margin-left: calc(-100vw / 2 + 100% / 2);
+		margin-right: calc(-100vw / 2 + 100% / 2);
+		max-width: 100vw;
+	}
+
+	.wp-block-image figure .alignwide {
+		margin-left: -100px;
```

Comment:
Use `rem()` instead of `px` whenever possible. The same comment applies to other usage of `px`.